jmbase/jmbase/support.py

```diff
@@ -124,3 +127,7 @@ def get_password(msg): #pragma: no cover
     if not isinstance(password, bytes):
         password = password.encode('utf-8')
     return password
+
+def print_jm_version(option, opt_str, value, parser):
+    print("JoinMarket " + JM_CORE_VERSION)
+    sys.exit(EXIT_SUCCESS)
```

Comment:
Seems a bit strange in that it's far from unlikely that we'd find a use for printing out the Joinmarket version without exiting? I realise this is a very minor point to question, we can always access JM_CORE_VERSION directly. But if this is specifically for scripts you could put that 'shutdown' feature in the script itself (or the code shared by scripts) rather than the code shared by everything, including daemon, GUI etc.

Reply:
OptionParser wants callback and it should be in jmbase, to not have duplication. Could be split into two functions, but I don't see a reason for that right now, and can be changed in future very easy, if needed.